app/src/main/res/values/colors.xml

```diff
@@ -11,4 +11,7 @@
     <color name="navy">#005c88</color>
     <color name="caribbean">#54BCEB</color>
     <color name="scarlet">#EF4135</color>
+    <color name="cancel_dialog_color">#f9f6e5</color>
```

Comment:
No need to create these specific colors, remove the 3, please. this can be reviewed together with the app design